Client/index.js

```diff
@@ -25,71 +25,56 @@ const game = document.getElementById('game');
 const Instructions = document.getElementById('directions');
 const x = document.createElement('video')
 
-setInterval(updateCountdown,1000);
+setInterval(updateCountdown, 1000);
 
-function updateCountdown(){
-   const min = Math.floor(time/60);
-   let sec = time % 60;
+function updateCountdown() {
+    const min = Math.floor(time / 60);
+    let sec = time % 60;
 
-   sec = sec < 10 ? '0'+ sec : sec;
+    sec = sec < 10 ? '0' + sec : sec;
 
 //    this sets conditions for timer so it stops on specified time interval
-   if(time >= 0){
-       timerEl.innerHTML =`${min} ${sec}`;
-       time--;
-   }else{
-    timerEl.innerHTML = "Time's up ";
-    game.style.display="none";
-    Instructions.style.display="none";
-    x.setAttribute('autoplay','autoplay');
-    // x.setAttribute('controls','controls');
-    x.setAttribute("src","/Vision.mp4");
-    x.setAttribute("width", "100%");
-    document.body.appendChild(x);
-
+    if (time >= 0) {
+        timerEl.innerHTML = `${min} ${sec}`;
+        time--;
+    } else {
+        timerEl.innerHTML = "Time's up ";
+        game.style.display = "none";
+        Instructions.style.display = "none";
+        x.setAttribute('autoplay', 'autoplay');
+        // x.setAttribute('controls','controls');
+        x.setAttribute("src", "/Vision.mp4");
+        x.setAttribute("width", "100%");
+        document.body.appendChild(x);
+
+    }
 }
- }
-     
-
 
 
 // This is the counter for the GETTISM words  
 
 
 const getty = document.querySelectorAll('.gettism')
 // const change =document.getElementsByTagName('i')
-const counter = document.querySelector('#counter')
+const counter = document.querySelector('#counter');
+const message = document.getElementById('message');
 let integer = 0
 
-for(i=0; i<getty.length; i++){
-
-    
-    getty[i].addEventListener('mouseover', ()=>{
+for (let i = 0; i < getty.length; i++) {
+    getty[i].addEventListener('mouseover', (e) => {
```

Comment:
Before you were using getty.style but getty is the actual array object you're looping through so doing it that way wouldn't work, you have to access the individual node itself. So I passed in an event parameter to access the target and update the display that way.